charts/chronicle/templates/tests/api-test.yaml

```diff
@@ -25,7 +25,7 @@ spec:
           command: [ "sh", "-c" ]
           args:
             - |
-              URL="{{ include "chronicle.id-provider.service.url" . }}"
+              URL="{{ include "chronicle.id-provider.service.jwks.url" . }}"
```

Comment:
This is okay for now, [CHRON-404](https://blockchaintp.atlassian.net/browse/CHRON-404) has been raised.